### PR TITLE
little things

### PR DESCRIPTION
**Pull request by xyzroe**
changed id model for LeTV build
changed number of keys for the LeTV build

### CODE REVIEW

**Comment by nurikk** [comment]
No need for model change, z2m already has converter.

**Comment by xyzroe** [comment]
how about right image for device and name?
like this:
image

**Comment by nurikk** [comment]
> how about right image for device and name?
> like this:
> image

it's useless feature, only supported in proprietary gateway. I wouldn't merge this

File: z2m-converter/FreePad_LeTV_8.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
+module.paths.push(require.main.path+ '/node_modules') //sovle Cannot find module 'zigbee-herdsman-converters'
```
Comment (nurikk) [inline z2m-converter/FreePad_LeTV_8.js:1]: Nope

**Comment by xyzroe** [comment]
Can an external converter for zigbee2mqtt correct your opinion?
image

**Comment by nurikk** [comment]
> Can an external converter for zigbee2mqtt correct your opinion?
> image

The whole idea of adding new converter just to change device pic is wrong, you're free to fork this repo and do whatever you want, but I won't merge this pr

**Comment by xyzroe** [comment]
OK

**Comment by xyzroe** [comment]
There is already a converter not only in the proprietary SLS, but also in the open source [zigbee-herdsman-converters](https://github.com/Koenkk/zigbee-herdsman-converters/pull/1769)
it's just a couple of lines of code with separate IF's, but FreePad releases for all hardware will be in one repository.
Can you change your mind?

**Pull request merged**